llarp/net/sock_addr.cpp

```diff
@@ -113,7 +115,13 @@ namespace llarp
     init();
 
     memcpy(&m_addr, &other, sizeof(sockaddr_in6));
-
+    if (other.sin6_addr.s6_addr[11] == 0xff and other.sin6_addr.s6_addr[10] == 0xff)
```

Comment:
for code clarity, probably should comment the significance of checking these two bytes.

Reply:
also it looks like this and the other `operator=` are almost identical.  some DRYness would go well here.

Reply:
dry'd out